bash/create_service_principal_docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
this script is using azure-cli docker image to create sp so that developers can skip the setup jobs such as install jq/azure-cli...